api/resolver/project/query.go

```diff
@@ -0,0 +1,23 @@
+package project
+
+import (
+	"context"
+	"github.com/robojones/graphql/prisma"
+)
+
+type Project struct {
+	Prisma *prisma.Client
+
+}
+
+func (resolver *Project) Members(ctx context.Context, project *prisma.Project) ([]prisma.User, error) {
+	return resolver.Prisma.Project(prisma.ProjectWhereUniqueInput{
```

Comment:
This would allow sql errors to be leaked.
Maybe it would be a good idea to check for sql errors (or generally for all non graphql errors) in a custom error handler, log them and replace them with a InternalError graphql error.

Reply:
The error handler solution would be best. For now, it's okay if the sql error message gets leaked (I mean in a rare case the user will see something like "xx would violate a unique constraint" or "connection to db lost" or whatever which is fine for now)